lib/simplecov/version.rb

```diff
@@ -1,22 +1,25 @@
 module SimpleCov
-  VERSION = "0.11.1"
-  def VERSION.to_a
+  version = "0.11.1"
```

Comment:
I think freezing the version constant is actually breaking the simplecov-teamcity-summary with older versions of rubygems. The simplecov-teamcity-summary has the following code:
`Gem::Version.new(SimpleCov::VERSION)`
and rubygems in version 1.8.23 has `@version.strip!` in its constructor. In newer versions of rubygems they're using `@version.strip` so I don't think it's an issue there.